site/pages/experimental-en.hbs

```diff
@@ -243,3 +256,46 @@ if ( s.getTime() &lt; msT && msT &lt; e.getTime() ) {
 		</div>
 	</div>
 </div>
+
+<h3 id="beta-alert">Beta alert designs</h3>
+<div class="provisional">
+	<details class="alert alert-danger">
```

Comment:
provisional css class name need to be sibling CSS of alert and not be the parent